aya/src/programs/utils.rs

```diff
@@ -51,3 +60,41 @@ pub(crate) fn find_tracefs_path() -> Result<&'static Path, ProgramError> {
         .as_deref()
         .ok_or_else(|| io::Error::new(io::ErrorKind::Other, "tracefs not found").into())
 }
+
+/// Get the program's load time.
+///
+/// The load time is specified by the kernel as nanoseconds since system boot,
+/// this function converts that u64 value into a human readable string.
+pub(crate) fn print_load_time(ns_since_boot: u64) -> String {
+    let time_boot = nix::time::clock_gettime(nix::time::ClockId::CLOCK_BOOTTIME).unwrap();
```

Comment:
Please use libc::clock_gettime over nix if we have to.

Reply:
e.g
```rust

pub fn time_since_boot() -> Duration {
    let mut time = libc::timespec {
        tv_sec: 0,
        tv_nsec: 0,
    };
    let ret = unsafe { libc::clock_gettime(libc::CLOCK_BOOTTIME, &mut time) };
    assert!(ret == 0);
    let tv_sec = time.tv_sec as u64;
    let tv_nsec = time.tv_nsec as u32;
    Duration::new(tv_sec, tv_nsec)
}

pub fn realtime() -> Duration {
    let mut time = libc::timespec {
        tv_sec: 0,
        tv_nsec: 0,
    };
    let ret = unsafe { libc::clock_gettime(libc::CLOCK_REALTIME, &mut time) };
    assert!(ret == 0);
    let tv_sec = time.tv_sec as u64;
    let tv_nsec = time.tv_nsec as u32;
    Duration::new(tv_sec, tv_nsec)
}
```

Reply:
Done!